frontend/src/scenes/paths/pathsLogic.ts

```diff
@@ -75,23 +75,29 @@ export const pathsLogic = kea<pathsLogicType<PathNode, PathResult>>({
                 const params = toParams({ ...filter, ...(refresh ? { refresh: true } : {}) })
 
                 const queryId = uuid()
-                const dashboardItemId = props.dashboardItemId as number | undefined
+                const dashboardItemId = props.dashboardItemId || props.fromDashboardItemId
```

Comment:
In line 51 of this file (and likewise in other insight type logics), should the key be set to this value as well?

Reply:
That's exactly the bug I'm trying to avoid. With the parameter in the URL, the `/insights` trends page thought it's displaying a dashboard item, and mounted the logic as such, so with the ID in the key. This made it that normal `trendsLogic().actions.anything()` just didn't work anymore.